src/seat/keyboard/mod.rs

```diff
@@ -12,18 +12,20 @@
 //! calloop event loop. Not doing so will prevent key repetition to work
 //! (but the rest of the functionnality will not be affected).
 
+#[cfg(feature = "calloop")]
+use calloop::{timer::Timer, RegistrationToken};
 #[cfg(feature = "calloop")]
 use std::num::NonZeroU32;
 #[cfg(feature = "calloop")]
 use std::time::Duration;
 use std::{
-    cell::RefCell,
+    cell::{Cell, RefCell},
```

Comment:
We could probably do something about the unusued imports?

Although like you said on Matrix, this really isn't important considering this code is going to be replaced soon.